collect_app/src/main/res/drawable/counter_minus_button_background.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="@color/color_on_surface_low_emphasis">
+    <item>
+        <shape android:shape="rectangle">
+            <corners
+                android:topLeftRadius="10dp"
```

Comment:
For this and the plus button, is there a shape appearance theme attribute we could use instead of a hardcoded one?

Reply:
I'm not sure what would work best but the [extended FAB](https://m3.material.io/components/extended-fab/overview) or [segmented buttons](https://m3.material.io/components/segmented-buttons/overview) have a similar design.

Reply:
`dialogCornerRadius` looks like a good fit here.